src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.skia.cs

```diff
@@ -181,7 +188,7 @@ private static IDisposable InnerCreateLayer(UIElement owner, LayoutState state)
 						area.Height - borderThickness.Top - borderThickness.Bottom);
 
 					var insideArea = new Rect(default, fullArea.Size);
-					var insertionIndex = 0;
+					// var insertionIndex = 0;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932478)